brozzler/cli.py

```diff
@@ -126,8 +127,38 @@ def configure_logging(args):
     )
 
 
+def mdfind(identifier):
+    try:
+        result = subprocess.check_output(
+            ["mdfind", f"kMDItemCFBundleIdentifier == {identifier}"], text=True
+        )
+    # Just treat any errors as "couldn't find app"
+    except subprocess.CalledProcessError:
+        return None
+
+    if result:
+        return result.rstrip("\n")
+
+
+def suggest_default_chrome_exe_mac():
+    # Try Chromium first, then Chrome
+    result = mdfind("org.chromium.Chromium")
+    if result is not None:
+        return f"{result}/Contents/MacOS/Chromium"
+
+    result = mdfind("com.google.Chrome")
+    if result is not None:
+        return f"{result}/Contents/MacOS/Google Chrome"
+
+
 def suggest_default_chrome_exe():
-    # mac os x application executable paths
+    # First ask mdfind, which lets us find it in non-default paths
+    if sys.platform == "darwin":
+        path = suggest_default_chrome_exe_mac()
+        if path is not None and os.path.exists(path):
+            return path
+
+    # Fall back to default paths if mdfind couldn't find it
```

Comment:
maybe the block beginning here should live in `suggest_default_chrome_exe_mac` too?

Reply:
Good call; moved all of that in.